draft-ietf-quic-transport.md

```diff
@@ -1590,7 +1590,7 @@ frames. A receiver MAY send timestamps for non-retransmittable packets.
 A receiver MUST not send timestamps in unprotected packets.
 
 A sender MAY intentionally skip packet numbers to introduce entropy into the
-connection, to avoid opportunistic acknowledgement attacks.  The sender MUST
+connection, to avoid opportunistic acknowledgement attacks.  The sender SHOULD
```

Comment:
It could also say "The sender MUST close the connection if a packet it knows is unsent is acknowledged."  Which keeps the must, but does not require unlimited state.